### PR TITLE
Update support info for 1.21

### PR DESCRIPTION
**Pull request by felicity-semmle**
This PR updates the support information based on the analysis change notes for 1.21.

The changes I'm aware of are:
* C/C++ - support for clang 8, gcc 8.3, and VS 2019, and preliminary support for armcc
* C# - support for C# up to 7.3. with .NET up to 4.8, Microsoft Visual Studio up to 2019, .NET Core up to 2.2, and support for C# 8.0 with .NET 3.0 preview features
* Java - support for Java 12

Preview: http://docteam.internal.semmle.com/felicity/sphinx-support-3/language-support.html

@Semmle/cpp, @Semmle/cs and @Semmle/java - please check that you're happy with these changes.

### CODE REVIEW

File: change-notes/support/language-support.rst [hunk 1]
```diff
@@ -13,7 +13,9 @@ Note that where there are several versions or dialects of a language, the suppor
 
 .. container:: footnote-group
 
-    .. [1] The best results are achieved with COBOL code that stays close to the ANSI 85 standard.  
-    .. [2] Builds that execute on Java 6 to 11 can be analyzed. The analysis understands Java 11 language features.
-    .. [3] JSX and Flow code, YAML, JSON, and HTML files may also be analyzed with JavaScript files. 
-    .. [4] TypeScript analysis is performed by running the JavaScript extractor with TypeScript enabled. This is the default for LGTM.   
+    .. [1] Support for the Arm Compiler (armcc) is preliminary.
+    .. [2] In addition, support for the preview features of C#8 and .NET Core 3.0.
```
Comment (calumgrant) [inline change-notes/support/language-support.rst]: space between C# and 8 ?

Comment (calumgrant) [reply change-notes/support/language-support.rst]: Sorry, it doesn't follow the C scheme of C89 C++11 etc.

Comment (felicity-semmle) [reply change-notes/support/language-support.rst]: Thanks. I'll update it to add the space shortly.

Comment (felicity-semmle) [reply change-notes/support/language-support.rst]: Should it be "C# 8.0"  or is "C# 8" okay?

**Comment by felicity-semmle** [comment]
@yh-semmle - please can you confirm whether you're happy with the change to "Java 12". After that, I think this PR is ready to merge.

**Pull request merged**